doc/source/structures/misc/config.rst

```diff
@@ -6,11 +6,35 @@ Configuration of kOS
 
 .. structure:: Config
 
-    :struct:`Config` is a special structure that allows your kerboscript programs to set or get the values stored in the kOS plugin's config file.
-
-    The options here can also be set by using the user interface panel shown here. This control panel is part of the :ref:`App Control Panel <applauncher>`
-
-    In either case, whether the setting is changed via the GUI panel, or via script code, these are settings that **affect the kOS mod in all saved games** as soon as the change is made. It's identical to editing the config file in the kOS installation directory, and in fact will actually change that file the next time the game saves its state.
+    :struct:`Config` is a special structure that allows your kerboscript
+    programs to set or get the values stored in the kOS plugin's configuration.
+    Some of the global values are stored in an external file, while save game
+    specific values are stored in that save file.
+
+    .. note::
+        .. versionadded:: v1.0.2
+            Prior to this version of kOS, all settings were stored globally in
+            a single external file.  KSP version 1.2.0 introduced a new way to
+            store settings within the save file itself, and most settings were
+            migrated to this system.
+
+    .. note::
+        If your save file has not yet migrated to the new settings storage
+        system and an old config file is present, you will be prompted with a
+        dialog box offering to migrate the old settings or use the defaults.
+        You may also choose to prevent further attempts to migrate settings,
+        which will instruct kOS to set the ``InstructionsPerUpdate`` to a
+        negative value.
```

Comment:
Our docs never explained anywhere that setting it to -1 is just how we record the fact that the settings are now defunct.  I think it makes more sense to mention that (that the settings are flagged as "do not use"), than mentioning the mechanism by which we do it and not explaining what that means.